game/script.js

```diff
@@ -181,10 +185,19 @@ var fallingElementsIntervalId = setInterval(() => {
                 life -= 1;
             }
         scoreBoard.innerText = score;
-        scoreLife.innerText = life;
         point.classList.remove('points', 'milk', 'mouse', 'drop', 'negative');
 
+        switch (life) {
+            case 2:
+                life1.remove();
+                break;
+            case 1:
+                life2.remove();
+                break;
+        }
+
         if (life <= 0) {
+            life3.remove();
             gameOver();
```

Comment:
Game over dziala, ale w przypadku gry gracz mial 2 zycia i zlapie bombe to wciaz wyswietlaja sie 2 glowy kocie

Reply:
U mnie działa ok. Niezależnie od tego czy bomba jest złapana przy wyniku 2 życia, czy 1 życie.

Reply:
U mnie też działa. Nie wiem zatem czy zmieniać. Sprawdź może jeszcze raz u siebie.